package.json

```diff
@@ -5,15 +5,19 @@
   "main": "lib/index.js",
   "scripts": {
     "build": "npm run clean && babel src --out-dir lib --ignore *-test.js",
+    "build:umd": "cross-env NODE_ENV=development webpack",
+    "build:umd:min": "cross-env NODE_ENV=production webpack",
     "clean": "rimraf lib",
-    "lint": "esw src --color",
+    "lint": "esw src webpack.config --color",
+    "lint:fix": "npm run lint -- --fix",
     "lint:watch": "npm run lint -- --watch",
     "prepublish": "npm run lint && npm run test && npm run build",
     "test": "mocha --compilers js:babel-register src/**/*-test.js",
     "test:watch": "npm run test -- --watch src/**/*-test.js"
   },
   "files": [
-    "lib"
+    "lib",
+    "dist"
```

Comment:
Are `build:umd` and `build:umd:min` part of the Travis build? If not, there will be no `dist/` to include, no?